res/Sectors/M1105/Avereguar.txt

```diff
@@ -0,0 +1,342 @@
+# Generated by https://travellermap.com
+# 2018-11-05T09:34:25+00:00
+
+# Avereguar
+# 7,3
+
+# Name: Avereguar (Hv)
+
+# Milieu: M1105
+
+# Credits: Avereguar sector was designed by Mark Humphreys.
+
+# Author:    Mark Humphreys
+
+# Subsector A: Aqaroo
+# Subsector B: Uafost
+# Subsector C: 
+# Subsector D: 
+# Subsector E: Lonylua
+# Subsector F: Avereguar
+# Subsector G: 
+# Subsector H: 
+# Subsector I: Itrasku
+# Subsector J: 
+# Subsector K: 
+# Subsector L: 
+# Subsector M: 
+# Subsector N: 
+# Subsector O: 
+# Subsector P: 
+
+# Alleg: H1: "Hive Federation Member"
+# Alleg: H2: "Hive Federation Member"
+# Alleg: H3: "Hive Federation Member"
```

Comment:
My data processing system complains there are several missing allegiances from this list but used in the data: 
* H4
* H5
* H6
* H?
* Hf

Reply:
I've added the descriptions of these to the files, it's to reflect the Federation style of government of the Hive Federation.